README.md

```diff
@@ -97,22 +116,19 @@ $ fh list releases nixos/nixpkgs
 ...
 ```
 
-### Listing organizations and flakes
+### Listing organizations, flakes, and versions
```

Comment:
super minor nit: should the order in this heading  be the same order as listed below? (flakes, orgs, versions)